[install error&bug] mac time_posix error

system info: macos 10.10.5
grpc version: grpc1.0

```
src/core/lib/support/time_posix.c: In function 'now_impl':
src/core/lib/support/time_posix.c:130:39: error: conversion to 'double' from 'uint64_t' may alter its value [-Werror=conversion]
       now_dbl = (mach_absolute_time() - g_time_start) * g_time_scale;
                                       ^
[C]       Compiling third_party/boringssl/crypto/x509/x509rset.c
[C]       Compiling third_party/boringssl/crypto/x509/x509spki.c
[C]       Compiling third_party/boringssl/crypto/x509/x509type.c
[C]       Compiling third_party/boringssl/crypto/x509/x_algor.c
[C]       Compiling third_party/boringssl/crypto/x509/x_all.c
[C]       Compiling third_party/boringssl/crypto/x509/x_attrib.c
[C]       Compiling third_party/boringssl/crypto/x509/x_crl.c
[C]       Compiling third_party/boringssl/crypto/x509/x_exten.c
cc1: all warnings being treated as errors
[C]       Compiling third_party/boringssl/crypto/x509/x_info.c
make: *** [/Users/dzh/github/grpc/objs/opt/src/core/lib/support/time_posix.o] Error 1
make: *** Waiting for unfinished jobs....
```

--- a/src/core/lib/support/time_posix.c
+++ b/src/core/lib/support/time_posix.c
@@ -127,7 +127,7 @@ static gpr_timespec now_impl(gpr_clock_type clock) {
       now.tv_nsec = now_tv.tv_usec * 1000;
       break;
     case GPR_CLOCK_MONOTONIC:
-      now_dbl = (mach_absolute_time() - g_time_start) * g_time_scale;
+      now_dbl = ((double)(mach_absolute_time() - g_time_start)) * g_time_scale;
       now.tv_sec = (int64_t)(now_dbl * 1e-9);
       now.tv_nsec = (int32_t)(now_dbl - ((double)now.tv_sec) * 1e9);
       break;

